src/export/node_export_visualization.rb

```diff
@@ -40,10 +40,28 @@ def perform(export_interface)
     end
 
     def color_static_groups(static_groups)
-      group_nr = 0
-      static_groups.reverse.each do |group|
-        color_group(group, group_nr)
-        group_nr += 1
+      # Create a set for each group containing the edges of the group
+      edge_groups = static_groups.map do |group|
+        edges = Set.new
+        group.each do |triangle|
+          triangle.edges.each do |edge|
+            edges.add(edge)
+          end
+        end
+        edges
+      end
+
+      edge_groups.each do |group|
+        group.each do |edge|
+          edge.link.material = Sketchup.active_model.materials['standard_material']
+        end
+      end
+
+      # No, only the intersections are colored differently (black)
```

Comment:
I guess you meant "now".
Also: maybe you could add that the intersections are the rotation axis?